source/faq.txt

```diff
@@ -24,13 +24,25 @@ What Is the Difference Between "connectTimeoutMS", "socketTimeoutMS" and "maxTim
      - Description
    * - connectTimeoutMS
      - 30000
-     - ``connectTimeoutMS`` is a `connection-setting <https://mongodb.github.io/node-mongodb-native/3.5/reference/connecting/connection-settings/>`_ option that sets the time in milliseconds to establish a connection to the MongoDB server before timing out.
+     - ``connectTimeoutMS`` is a :doc:`connection option
+       </fundamentals/connection#connection-options>` that sets the time in
+       milliseconds for an individual connection from your connection pool to
+       establish a TCP connection to the MongoDB server before timing out.
+       Use the ``serverSelectionTimeoutMS`` option instead to fail when
+       :node-api:``MongoClient.connect <MongoClient.html#.connect>` is unable
+       to establish a connection with the MongoDB server.
    * - socketTimeoutMS
      - 360000
-     - ``socketTimeoutMS`` is a ``connection-settings`` option that sets the number of milliseconds a socket stays inactive after the driver has successfully connected before closing the connection.
+     - ``socketTimeoutMS`` is a :doc:`connection option
+       </fundamentals/connection#connection-options>` that sets the number of
+       milliseconds a socket stays inactive after the driver has successfully
+       connected before closing the connection.
    * - maxTimeMS
      - N/A
-     - :node-api:`maxTimeMS() </Cursor.html#maxTimeMS>` is a :manual:`cursor method </reference/method/js-cursor>` that specifies the max time limit for processing an operation on a cursor. If an operation runs over the time allotted it will return a timeout error.
+     - :node-api:`maxTimeMS() </Cursor.html#maxTimeMS>` is a method that
```

Comment:
a few things here:
- `maxTimeMS` is a connection option, cursor option, as well as something you can set per-operation (through `options`) 
- "`maxTimeMS` is the maximum amount of time the server should wait for an operation to complete, once it has reached the server. If an operations runs over the specified time, it will return a timeout error"